core/dbsecurity/auth/inc/authEvents.h

```diff
@@ -0,0 +1,89 @@
+//******************************************************************************
+// @@@ START COPYRIGHT @@@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+// @@@ END COPYRIGHT @@@
+//******************************************************************************
+#ifndef INCLUDE_AUTHEVENT_H
+#define INCLUDE_AUTHEVENT_H  1
+#include "common/evl_sqlog_eventnum.h"
+#include "CommonLogger.h"
+#include <string>
+#include <vector>
+
+// From a web search, the log message max length is a bit over 8000 bytes.
+// For DBSecurity don't believe any messages will be more than 1M
+#define MAX_EVENT_MSG_SIZE 1024
+
+// The ported code had the caller sending in the filename and line number
+// for certain events.  This has not been implemented at this time.
+struct AuthEvent
```

Comment:
This is fine as-is. However, since you've defined accessor methods, you're half-way to making this a class. Maybe go the rest of the way?